packages/portal/tests/unit/components/item/ItemMediaThumbnail.spec.js

```diff
@@ -0,0 +1,37 @@
+import { createLocalVue } from '@vue/test-utils';
+import { shallowMountNuxt } from '../../utils';
+import ItemMediaThumbnail from '@/components/item/ItemMediaThumbnail';
+// import sinon from 'sinon';
+
+const localVue = createLocalVue();
+
+const factory = ({ propsData = {}, mocks = {} } = {}) => shallowMountNuxt(ItemMediaThumbnail, {
+  localVue,
+  attachTo: document.body,
+  propsData,
+  mocks: {
+    $route: { query: {} },
+    $t: (key) => key,
+    ...mocks
+  }
```

Comment:
`$route` is not used in the component ATTOW. And there are never any mocks passed in this file's tests.

Reply:
Since the refactor for the thumbnail to be a link, i've added some more specs, some of those now use the route.